frontend/src/App.tsx

```diff
@@ -121,34 +121,35 @@ function TopBar() {
       <ThemeChangeButton />
       <IconButton
         aria-label="User menu"
-        aria-controls="menu-appbar"
+        aria-controls="customized-menu"
         aria-haspopup="true"
         onClick={handleMenu}
         color="inherit"
       >
         <Icon icon={accountIcon} />
       </IconButton>
-      <Menu
```

Comment:
I would assume that `Menu` resorts to a known element in the DOM. If we do this in many places that's meh..

Reply:
Menu gives no DOM element when it's not there unfortunately.